opengrok-indexer/src/main/java/org/opengrok/indexer/web/SearchHelper.java

```diff
@@ -469,7 +469,8 @@ private void getSuggestion(Term term, IndexReader ir,
             Suggestion s = new Suggestion(proj);
             try {
                 if (!closeOnDestroy) {
-                    SuperIndexSearcher searcher = RuntimeEnvironment.getInstance().getIndexSearcher(proj);
+                    SuperIndexSearcher searcher;
+                    searcher = RuntimeEnvironment.getInstance().getIndexSearcher(proj);
```

Comment:
what is the benefit in this change?

Reply:
ah, that's a leftover from earlier modification.